rallyrolebot/cogs/rally_cog.py

```diff
@@ -95,3 +97,11 @@ async def unset_rally_id(self, ctx, rally_id):
     @validation.owner_or_permissions(administrator=True)
     async def admin_unset_rally_id(self, ctx, discord_id: discord.User, rally_id):
         data.remove_discord_rally_mapping(discord_id, rally_id)
+
+    @commands.command(name="coinlink", help="To generate a custom coin link, type $coinlink <CoinName> <COIN/USD> <Amount> <Memo>")
+    async def generate_coinlink_deeplink(self,ctx, coin: Union[CreatorCoin, CommonCoin], currencyType:CurrencyType, amount:int, memo:str):
+
+        params = {"inputType":currencyType, "amount":amount, "note":memo}
+        deeplink = 'https://www.rally.io/creator/' + coin["symbol"] + "/?" + urlencode(params)
+
+        await ctx.send(deeplink)
```

Comment:
Please use `utils.pretty_print()` so that we have a consistent look and feel across all outputs from RallyRoleBot.